tests/istio.io/README.md

```diff
@@ -6,12 +6,66 @@ This folder contains tests for the different examples on Istio.io.
 
 ## Writing Tests
 
+Tests in the Istio.io framework support the following test types:
```

Comment:
This section needs examples of each type of test and each type of output. Using examples, we doesn't only make things clearer to new test writers, it would also allow us to show what quality we expect. I would suggest using examples that showcase testing best practices.

```suggestion
The Istio.io test framework supports the following test types:
```

Reply:
Currently the outputtype only changes the extension on the output file. Should we still put examples of each output type?